packages/cog/src/cli/actions/action.cog.ts

```diff
@@ -69,6 +65,8 @@ export class ActionCogCreate extends CommandLineAction {
 
         const isCommit = this.commit?.value ?? false;
 
+        const resampleMethod = this.resampleMethod?.value ?? 'cubic';
```

Comment:
If this was in the job.json then we not need to pass this in here. We should put this into the job.json.